docs_theme/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "drf_docs_theme",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack build",
+    "build:production": "webpack build --mode=production",
+    "build:watch": "webpack build --watch"
```

Comment:
When I try to build the docs on master, I run `mkdocs build` and then I can just open the `site/index.html` file and it just works. 

With this branch, the html file is missing all CSS styles, until I run `npm i && npm run build`, which seems like an extra step needed to deploy the docs. I have no idea how the actual docs are currently deployed nor if the project is willing to change that, but that might be a deal breaker.

Reply:
Instead of downloading the css files from bootstrap directly I've  imported the needed parts of bootstrap manually which decreases the final bundle size and improves performance. We could build the final css files locally and commit them to the repo or create a workflow to automatically build the final css files both solutions work. In my opinion the second solution is better and cleaner.